coordinates/w32h32.json.example

```diff
@@ -225,6 +227,14 @@
       "games_back": {
         "x": 64
       }
+    },
+    "postseason": {
+      "matchup_y_gap": 12,
+      "series_x_gap": 32,
+      "wc_x_start": 3,
+      "wc_y_start": 12,
+      "ds_a_y_start": 51,
+      "__comment": "all other coords are based off wild card position"
```

Comment:
Open to suggestions on this... Didn't look, but seems like if this is here then postseason brackets render, so may need to add an explicit `enable`/`disable` flag